components/style/values.rs

```diff
@@ -1317,6 +1319,17 @@ pub mod specified {
         }
     }
 
+    #[derive(PartialEq, Clone, Debug)]
+    #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+    pub struct UrlExtraData {
```

Comment:
Why not moving it to gecko_values.rs? That would allow you to skip all the `cfg`s